src/js/helpers/single_cookie.js

```diff
@@ -0,0 +1,87 @@
+"use strict";
+
+var single_cookie = null;
+var data = require('../data'),
+    default_cookies = require('./cookies');
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    if( ! single_cookie ) {
+      this.load();
+    }
+    // Don't break the build
+    domain = '';
+    excl_subdomains = false;
+
+    // Set an expiration for the values
+    if( name === data.containers.session ) {
+      value += data.delimiter + data.aliases.single_expire + '=' + (Math.floor(Date.now() / 1000) + minutes * 60);
+    }
+    single_cookie[default_cookies.unsbjs(name)] = value;
+  },
+
+  get: function(name) {
+    if( ! single_cookie ) {
+      this.load();
+    }
+
+    // Simulate checking session expiration
+    if( name === data.containers.session ) {
+      var session = single_cookie[default_cookies.unsbjs(name)];
+      if( session ) {
+        var sts = session.match(/sts=(\d+)/);
```

Comment:
❓ I think this should be `sxp`, as per `data.aliases.single_expire`?

```suggestion
        var sxp = session.match(/sxp=(\d+)/);
```